src/server/game/DungeonFinding/LFGMgr.cpp

```diff
@@ -2148,12 +2148,15 @@ namespace lfg
         {
             error = LFG_TELEPORTERROR_COMBAT;
         }
+        else if (!sWorld->getBoolConfig(CONFIG_LFG_TELEPORT) && !m_Testing)
```

Comment:
This should only trigger if debug lfg is not active and the config is not active.